src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -33,34 +40,73 @@ public AddressBook(ReadOnlyAddressBook toBeCopied) {
         persons.setPersons(toBeCopied.getPersonList());
     }
 
-    //// list overwrite operations
+    //// undo & redo operations
 
     /**
-     * Makes a copy of persons, and stores it in personsList.
+     * Empties the redoList.
      */
-    private void save() {
+    private void clearRedoList() {
```

Comment:
This method seems to be redundant as there is only one call that directly use `redoStack.clear()`.